src/random.c

```diff
@@ -245,24 +245,22 @@ accum exponential_dist_variate(
     accum A = 0.0k;
     uint32_t U, U0, USTAR;
 
-outer:
-    U = uni_rng(seed_arg);
-    U0 = U;
-
-inner:
-    USTAR = uni_rng(seed_arg);
-    if (U < USTAR) {
-	return A + (accum) ulrbits(U0);
+    while (true) {
```

Comment:
needs realigning, as mixture of tabs vs spaces on my system makes these look like they are indented at the same level. github itself changes tab and spaces so that it looks indented

Reply:
@dkfellows